operator/src/main/resources/scripts/introspectDomain.sh

```diff
@@ -48,6 +48,19 @@ trace "Introspecting the domain"
 
 env | tracePipe "Current environment:"
 
+#
+# Define helper fn to create a folder
+#
+
+function createFolder {
+  mkdir -m 750 -p $1
+  if [ ! -d $1 ]; then
+    trace "Unable to create folder $1"
```

Comment:
Add SEVERE into this statement.   `trace SEVERE "Unable to create folder $1"`

Reply:
createFolder() function exists in utils.sh so removed this duplicate in introspectDomain.sh